int32[]   motor_id
float32[] position
float32[] velocity
